lib/io/dart_io.cc

```diff
@@ -13,9 +13,12 @@ using tonic::ToDart;
 namespace blink {
 
 void DartIO::InitForIsolate() {
-  DART_CHECK_VALID(Dart_SetNativeResolver(Dart_LookupLibrary(ToDart("dart:io")),
-                                          dart::bin::LookupIONative,
-                                          dart::bin::LookupIONativeSymbol));
+  Dart_Handle result = Dart_SetNativeResolver(
+      Dart_LookupLibrary(ToDart("dart:io")), dart::bin::LookupIONative,
+      dart::bin::LookupIONativeSymbol);
+  if (Dart_IsError(result)) {
+    Dart_PropagateError(result);
```

Comment:
Isn't this called from the isolate create callback. The error handling contract there is to return nullptr from the callback as the isolate handle. In that case, I think this function should return a bool or something to indicate failure and have the create callback return null.

Reply:
Right, this code is still incorrect, but it will unblock the next Dart roll.